packages/react-charting/src/components/VerticalStackedBarChart/VerticalStackedBarChart.base.tsx

```diff
@@ -32,6 +32,7 @@ import {
   getTypeOfAxis,
```

Comment:
Make the change for Grouped Vertical Bar chart as well.

Reply:
done